setup.sh

```diff
@@ -446,6 +447,30 @@ function install_home()
   remove_legacy_git_from_kernel_config_manager
   # Update version based on the current branch
   update_version
+  # Show current environment in terminal
+  show_current_env
+}
+
+function show_current_env()
```

Comment:
I still don't like this function name because this is more like a setup than an action like 'show'... please, find a better name for that... maybe look for some ideas from the git project.

Reply:
How about _setup_bashrc_current_env_custom()_ ?

Reply:
Maybe `setup_bashrc_to_show_current_kw_env`